TrigonGithubHanuka2018/src/org/usfirst/frc/team5990/robot/commands/CollectIn.java

```diff
@@ -1,48 +1,46 @@
-/*----------------------------------------------------------------------------*/
-/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
-/* Open Source Software - may be modified and shared by FRC teams. The code   */
-/* must be accompanied by the FIRST BSD license file in the root directory of */
-/* the project.                                                               */
-/*----------------------------------------------------------------------------*/
-
 package org.usfirst.frc.team5990.robot.commands;
 
-import edu.wpi.first.wpilibj.command.Command;
 import org.usfirst.frc.team5990.robot.Robot;
 
+import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj.command.Command;
+
 /**
- * An example command.  You can replace me with your own command.
+ *
  */
-public class ExampleCommand extends Command {
-	public ExampleCommand() {
+public class CollectIn extends Command {
+
```

Comment:
Change name of class to be more suitable (the command isn't always "in")
maybe "Collect"?